src/Action/AddAction.php

```diff
@@ -50,10 +51,8 @@ class AddAction extends BaseAction
         'viewVar' => null,
         'relatedModels' => true,
         'entityKey' => 'entity',
-        'saveOptions' => [
-            'validate' => true,
-            'atomic' => true
-        ],
+        'entityOptions' => [],
+        'saveOptions' => [],
```

Comment:
Well, all the work I've done in the marshaller and save have been to encourage people to use a single array for options to avoid duplication and to also drive some automation out of it.